code/game/machinery/machinery.dm

```diff
@@ -284,6 +284,7 @@ Class Procs:
 
 //set_machine must be 0 if clicking the machinery doesn't bring up a dialog
 /obj/machinery/attack_hand(mob/user, check_power = 1, set_machine = 1)
+	..() // unbuckling etc

```

Comment:
in most places like this, iirc, it's more like `if(..()) return 1`
maybe you should look at other unbuckling stuff to make sure? I could be wrong